Bump pytest from 7.2.0 to 8.0.1 (#15)
IBM/detect-secrets-stream: docs/detect-secrets-suite-faq.md

> # Welcome to `detect-secrets-suite`!
> 
> This documentation is intended for GitHub Enterprise end users. The rest of the documentation in this repository is intended for developers managing the Detect Secrets Stream tool.

Reviewer: ```suggestion
# Detect Secrets Stream FAQ

This documentation is intended for GitHub Enterprise end users.
```

### My push has timed out, what should I do?

If you're seeing a message such as:

```shell
remote: Push info collect pre-receive hook failed to finish within 3 seconds with error code 124
remote: push_info.sh: execution exceeded 5s timeout
To <REDACTED>/secret-corpus-db !
[remote rejected] gen-db-tool -> gen-db-tool (pre-receive hook declined)
```

Although the pre-receive hook should not fail by design, there have been rare instances where it does. If so, the development team will be aware that it has timed out; There is no need to report this.

Please re-attempt the push unaltered. If it continues to fail, please open a GitHub issue.

### My push has slowed down, what should I do?

There should be minimal delay to your push introduced by triggering the `detect-secrets` asynchronous scan. If this is not the case for you, please open a GitHub issue.

### My push was made but my CI / CD didn't trigger or execute?

Please take care to distinguish between the `detect-secrets-suite` service and other CI/CD.
e.g.:

- Your organization / repository's CI/CD
- CI/CD secret checking
- GitHub Apps
- Pre-commit hooks you have installed locally

> For your own CI/CD issues, please refer to your GitHub Organization owner.

Reviewer: Let's take out this one, does not seem too relevant.

Reviewer 2: https://github.com/IBM/detect-secrets-stream/pull/15/commits/db95d2ecee4ece50dc5b922b4774d936c5e6132a
